catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -387,19 +442,23 @@ private static void addExtentResourceSizeAttribute(
       List<String> addedAttributes,
       Node fileNode,
       String attribute) {
-    if (shouldAdd(buildAttr(attribute, NsiliConstants.EXTENT), resultAttributes)
-        && metacard.getResourceSize() != null) {
-      try {
-        Double resSize = Double.valueOf(metacard.getResourceSize());
-        Double resSizeMB = convertToMegabytes(resSize);
-        if (resSizeMB != null) {
-          addDoubleAttribute(graph, fileNode, NsiliConstants.EXTENT, resSizeMB, orb);
-          addedAttributes.add(buildAttr(attribute, NsiliConstants.EXTENT));
+    if (shouldAdd(buildAttr(attribute, NsiliConstants.EXTENT), resultAttributes)) {
+      Double resSizeMB = 0.0;
+      if (metacard.getResourceSize() != null) {
+        try {
+          Double resSize = Double.valueOf(metacard.getResourceSize());
+          resSizeMB = convertToMegabytes(resSize);
+          if (resSizeMB == null) {
+            resSizeMB = 0.0;
+          }
+        } catch (NumberFormatException nfe) {
+          LOGGER.debug(
+              "Couldn't convert the resource size to double: {} - defaulting to 0.0",
```

Comment:
Might be better to indicate couldn't convert to megabytes, and also log the exception with the message

Reply:
It's not the conversion to megabytes that's the issue, but rather the parsing the String. Will update the message to clarify